kernel/data-pipeline/dialect/opengauss/src/main/java/org/apache/shardingsphere/data/pipeline/opengauss/ingest/wal/decode/MppdbDecodingPlugin.java

```diff
@@ -91,6 +97,21 @@ private AbstractWALEvent decodeDataWithTX(final String dataText) {
         return result;
     }
     
+    private AbstractWALEvent decodeVersionGreaterThan3(final String dataText) {
+        AbstractWALEvent result = new PlaceholderEvent();
+        if (dataText.startsWith("BEGIN")) {
+            int beginIndex = dataText.indexOf("CSN:") + "CSN:".length() + 1;
+            int endIndex = dataText.indexOf("first_lsn") - 1;
+            result = new BeginTXEvent(null, Long.parseLong(dataText.substring(beginIndex, endIndex)));
+        } else if (dataText.startsWith("commit")) {
```

Comment:
It's better to compatible with `COMMIT`, since it's different as `BEGIN`, though it's defined as `commit` in document.
e.g. Use `dataText.startsWith("commit") || dataText.startsWith("COMMIT")`